src/analysis/retail/evoker/preservation/modules/talents/EmeraldBlossom.tsx

```diff
@@ -98,6 +145,56 @@ class EmeraldBlossom extends Analyzer {
     );
   }
 
+  get guideSubsection(): JSX.Element {
+    const styleObj = {
+      fontSize: 20,
+    };
+    const styleObjInner = {
+      fontSize: 15,
+    };
+    const explanation = (
+      <p>
+        <SpellLink spell={SPELLS.EMERALD_BLOSSOM} /> is the core spell in the Blossom talent build
+        for Preservation Evokers. It is critical to cast{' '}
+        <SpellLink spell={SPELLS.EMERALD_BLOSSOM} /> on targets that are nearby other allies (more
+        often than not this is a melee) so that you can fully utilize its healing, while trying to
+        not overheal in the process.
+      </p>
+    );
+    const data = (
+      <div>
+        <RoundedPanel>
+          <div>
+            <strong>
+              <SpellLink spell={SPELLS.EMERALD_BLOSSOM_CAST} /> casts
+            </strong>{' '}
+            <small>
+              {' '}
+              - Blue is a perfect cast with {this.perfectThreshold} targets hit, Green is a good
+              cast with {this.goodThreshold} or more with moderate to low overhealing, Yellow is an
+              ok cast with a moderate amount of overheal, and Red is a bad cast with high overheal
+              or few targets hit. This graphic does not include{' '}
+              <SpellLink spell={TALENTS_EVOKER.FIELD_OF_DREAMS_TALENT} /> due to their inherent
+              randomness.
+            </small>
+            <PerformanceBoxRow values={this.castEntries} />
+          </div>
+          <div style={styleObj}>
+            <small style={styleObjInner}>
+              <SpellLink spell={SPELLS.EMERALD_BLOSSOM} /> -{' '}
+            </small>
+            <strong>{this.averageNumTargets.toFixed(2)}</strong>{' '}
```

Comment:
Bit of a nit but this avg target calc includes field procs even though the Performance Box Row filters them out yes? I would say either get the filtered avg targets or don't show this on the graphic